pallets/loans/src/lib.rs

```diff
@@ -547,6 +625,118 @@ pub mod pallet {
             Ok(().into())
         }
 
+        /// Add reward for the pallet account.
+        ///
+        /// - `amount`: Reward amount added
+        #[pallet::weight(T::WeightInfo::add_reward())]
+        #[transactional]
+        pub fn add_reward(
+            origin: OriginFor<T>,
+            amount: BalanceOf<T>,
+        ) -> DispatchResultWithPostInfo {
+            let who = ensure_signed(origin)?;
+            ensure!(!amount.is_zero(), Error::<T>::InvalidAmount);
+
+            let reward_asset = T::RewardAssetId::get();
+            let pool_account = Self::account_id();
```

Comment:
We should use another pool account for this one is already used by the HKO market.